packages/config/config/devices/0x0330/VES-ZW-HLD-016.json

```
@@ -0,0 +1,127 @@
+{
+	"manufacturer": "Vesternet",
+	"manufacturerId": "0x0330",
+	"label": "VES-ZW-HLD-016",
+	"description": "High Load Switch",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0xd019"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "2",
+			"$import": "~/templates/master_template.json#state_after_power_failure_off_on_prev"
+		},
+		{
+			"#": "3",
+			"label": "Send Basic Report on State Change",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"defaultValue": 1
+		},
+		{
+			"#": "4",
+			"label": "Over Current Protection",
+			"description": "Triggered at over 16A.",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"defaultValue": 1
+		},
+		{
+			"#": "5",
+			"label": "Switch Type",
+			"valueSize": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Momentary",
+					"value": 0
+				},
+				{
+					"label": "Toggle",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "10",
+			"label": "Power Report Absolute Threshold",
+			"description": "Report when power changes by W value, unit is 1W.",
+			"unit": "W",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"unsigned": true,
+			"defaultValue": 5,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "11",
+			"label": "Current Report Absolute Threshold",
+			"description": "Report when current changes by A value, unit is 0.1A.",
+			"unit": "0.1A",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"unsigned": true,
+			"defaultValue": 1,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "12",
+			"label": "Voltage Report Absolute Threshold",
+			"description": "Report when voltage changes by V value, unit is 1V.",
+			"unit": "V",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"unsigned": true,
+			"defaultValue": 1,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "13",
+			"label": "Report Interval",
```

```suggestion
			"label": "Energy/Power Report Interval",
```
and remove the description please.

---

Should this be "Energy/Power/Voltage/Current Report Interval" since it covers all of them?  Just conscious that removing the description makes it less clear what Parameter covers?

---

I'd be fine with that. We just try to keep labels short and typically devices report all they can on an interval.